src/lgr_basic/forms.py

```diff
@@ -37,9 +36,8 @@ def clean(self):
         return cleaned_data
 
     def clean_labels(self):
-        rz_lgr = self.cleaned_data['rz_lgr']
-        lgr_info = LGRInfo(rz_lgr, lgr=get_by_name(rz_lgr, with_unidb=True))
```

Comment:
@j-bernard I test that the same versions of unicode was passed for my use cases but I am not sure if with_unidb does more work than I expect

Reply:
I'm not sure you would get unicode information in all cases, but maybe in that specific case where you would only have rz lgr it will be ok